src/core_plugins/kibana/public/dashboard/saved_dashboard/saved_dashboard.js

```diff
@@ -67,6 +67,8 @@ module.factory('SavedDashboard', function (courier, config) {
     }
   };
 
+  SavedDashboard.fieldOrder = ['title', 'description'];
```

Comment:
In the context of this PR I can totally understand why we only want to list `title` and `description` here: after all, those are the only fields we care about sorting to the top today. However, I'm thinking ahead to reading this line of code (and its cousins in other saved object files) several months into the future. I think there's a chance of a developer getting confused then about why only these two fields are in this list when there are clearly other fields in this saved object as well. Also if we wanted to add another field at position 5 (for instance) in the future, we would need to explicitly add the fields at positions 3 and 4 too at that time. 

So consider adding a comment above this line (and its counterparts in other saved object files) explaining why only some fields need to be listed here.